.chronus/changes/filter-readonly-parameters-2025-11-10.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: breaking
+packages:
+  - "@azure-tools/typespec-client-generator-core"
+---
+
+Filter ReadOnly parameters from `SdkServiceMethod` parameters. ReadOnly parameters (those with `@visibility(Lifecycle.Read))` are now correctly excluded from method signatures since they cannot be set by the user.
```

Comment:
```suggestion
Filter read-only parameters from `SdkServiceMethod` parameters. Read-only parameters (those with `@visibility(Lifecycle.Read))` are now correctly excluded from method signatures since they cannot be set by the user.
```